mod_wsbridge/mod_wsbridge.c

```diff
@@ -700,6 +832,25 @@ wsbridge_callback_ws(struct lws *wsi, enum lws_callback_reasons reason,
 			tech_pvt->has_dtmf = FALSE;
 		}
 		switch_mutex_unlock(tech_pvt->dtmf_mutex);
+
+		{
+			void* pop;
+			if (Queue_pop(&tech_pvt->eventQueue, &pop) == SWITCH_STATUS_SUCCESS) {
+				cJSON *json_message;
+
+				json_message = cJSON_Parse((char *)pop);
+				if (json_message) {
+					// control json event
+					on_event(tech_pvt, json_message);
+					// send json
+					add_content_type(tech_pvt, json_message);
+					send_bugfree_json_message(wsi, json_message);
+				}
+				cJSON_Delete(json_message);
+				switch_safe_free(pop);
+			}
+		}
+
```

Comment:
Instead of having a block of code inline, let's factor it out into a new function
`void wsbridge_process_message(private_t *tech_pvt, struct lws *wsi){}`  which encapsulates popping the event queue in the callback.  It's similar to its counterpart channel_process_message which encapsulates pushing a message on the event queue.